src/shogun/classifier/mkl/MKL.h

```diff
@@ -95,6 +95,8 @@ class CMKL : public CSVM
 		 */
 		virtual ~CMKL();
 
+		CSGObject* clone();
```

Comment:
Was thinking about that, too. I don't know how you guys want to go about this in the future. If you want to make use of this feature in future changes, this implies changing more than needed for a fix in many cases, since compilers will give warnings when `override` is used inconsistently within one unit.